frontend/frontend/src/components/Timer.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div>
+    <span>{{ time }}</span>
+  </div>
+</template>
+
+<script>
+export default {
+  mounted() {
+    this.play();
+  },
+  props: ["state"],
+  data() {
+    return {
+      time: "00:00:00",
+      startTime: null,
+      endTime: null,
+      runTime: 0,
+      started: null,
+      running: false,
+    };
+  },
+  watch: {
+    state() {
```

Comment:
We definitely need to watch for a state change. But I propose changing this to use a handler and setting the immediate value to true so that the function play/end runs immediately after a state change is perceived. As of now, the play function only plays when it's mounted which won't work when I try to change the timer state to "stopped"

Reply:
@CarlaLeal The watch function will be reactive to state changes, it just need to be properly integrated with the game logic to have the state changed by game.vue, the values of the state variable could be changed to a boolean or either running or not running as the timer is only mounted in a game.

Reply:
> We definitely need to watch for a state change. But I propose changing this to use a handler and setting the immediate value to true so that the function play/end runs immediately after a state change is perceived. As of now, the play function only plays when it's mounted which won't work when I try to change the timer state to "stopped"

I've implemented your requested changes and submitted for re-review. If all looks good to you, then feel free to merge to main.